File: VB/WinSolution.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.BaseImpl

Namespace WinSolution.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal session As Session, ByVal currentDBVersion As Version)
			MyBase.New(session, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Using uow As New UnitOfWork(Session.DataLayer)
				Dim master As New Master(uow)
				master.Name = "Master"
				For i As Integer = 0 To 9
					Dim child As New Child(uow)
					child.Master = master
					child.Name = "Child " & i.ToString()
				Next i
				uow.CommitChanges()
			End Using
		End Sub
	End Class
End Namespace

File: CS/WinSolution.Module/Updater.cs
using DevExpress.ExpressApp;
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;

namespace WinSolution.Module {
    public class Updater : ModuleUpdater {
        public Updater(ObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
                Master master = ObjectSpace.CreateObject<Master>();
                master.Name = "Master";
                for (int i = 0; i < 10; i++) {
                    Child child = ObjectSpace.CreateObject<Child>();
                    child.Master = master;
                    child.Name = "Child " + i.ToString();
                }
                ObjectSpace.CommitChanges();
        }
    }
}
